src/utils/DMRoomMap.js

```diff
@@ -32,6 +41,34 @@ export default class DMRoomMap {
         }
     }
 
+    /**
+     * Returns a shared instance of the class
+     * that uses the singleton matrix client
+     * The shared instance must be started before use.
+     */
+    static shared() {
+        if (!DMRoomMap._sharedInstance) {
+            DMRoomMap._sharedInstance = new DMRoomMap(MatrixClientPeg.get());
```

Comment:
Good catch. I've made an explicit method to replace the shared instance, since this then leave start() and stop() open to being used to start & stop a non-shared instance. Seem OK?